lib/ama_layout/navigation.yml

```diff
@@ -19,13 +19,17 @@ member:
     link: "<%= Rails.configuration.driveredonline_site %>"
     icon: "fa-car"
   - text: "My Reward Dollars"
-    link: "<%= Rails.configuration.youraccount_site %>/reward_dollars"
+    link: "<%= Rails.configuration.youraccount_site %>/rewards"
```

Comment:
This link should be removed and replaced with a simple "#" - the top level link is only supposed to expand the menu, as far as I understand - check with @kaytcampbell to confirm